crates/aptos-faucet/core/src/funder/common.rs

```diff
@@ -467,3 +419,70 @@ impl GasUnitPriceManager {
             .gas_estimate)
     }
 }
+
+#[derive(Clone, Debug, Deserialize, Serialize, Parser)]
+pub struct AssetConfig {
+    /// Address of the account to send transactions from. On testnet, for
+    /// example, this is a550c18. If not given, we use the account address
+    /// corresponding to the given private key.
+    pub mint_account_address: Option<AccountAddress>,
```

Comment:
This belongs back in mint.rs, it's specific to minting, it's not common. Like the TransferFunder doesn't do anything with this. Consider refactoring the config so the truly common parts stay here but the mint specific stuff is in mint.rs.